src/isaw.policy/src/isaw/policy/profile.zcml

```diff
@@ -410,4 +410,14 @@
       handler=".setuphandlers.migrate_publication_title_fields"
       />
 
+  <genericsetup:upgradeDepends
+      sortkey="11"
+      source="39"
+      destination="40"
+      title="Remove PressRoom"
```

Comment:
Not super important, but this should probably be "Define 'social' image size". Also `sortkey` has no effect here.

Reply:
Indeed some truly lazy copy/pasting on my part.